src/main/resources/yamlschema/PrintResponse.yaml

```diff
@@ -0,0 +1,38 @@
+#version: 0.0.3
+title: PrintResponse
+type: object
+description: 'Represents a print response, status update for a single print request'
+properties:
+  requestId:
+    type: string
+    format: uuid
+    description: The request identifier of the print request
+    example: 14ab70e8-bd3b-400f-bd95-246caf9e4810
+  timestamp:
+    type: string
+    format: date-time
+    description: The date time the Voter Authority Certificate print request was updated to this status.
+    example: '2022-06-01T14:23:03.000Z'
+  statusStep:
+    type: string
+    description: The current print step this update relates to
+    enum:
+      - PROCESSED
+      - IN-PRODUCTION
+      - DISPATCHED
+      - NOT-DELIVERED
```

Comment:
Should there be a `DELIVERED`?  (I assume `PROCESSED` is before going into production - like accepted for printing)

Reply:
I think `DISPATCHED` means delivered (effectively). We wont know (and nor will the print provider) that its actually been delivered, unless the postman tells us. I dont think we're doing recorded delivery or similar, so once its been `DISPATCHED` thats the last anyone knows of it ...... except ....

... in the case that the postie can't deliver it, then it gets returned to sender, which is the print provider. In that case the print provider send us another status update saying `NOT-DELIVERED`